charger/twc3.go

```diff
@@ -161,12 +163,32 @@ func (v *Twc3) ChargingTime() (time.Duration, error) {
 	return time.Duration(res.SessionS) * time.Second, err
 }
 
+// Determine if workaround needed for TWC3 single phase vitals
+// This can only be determined while vehicle is charging
+func IsSplitPhase(v *Twc3, res Vitals) (bool) {
+	// Use anomalous phase C readings to identify, i.e.
+	//   "voltageA_v": 241.5,
+	//   "voltageB_v": 241.5,
+	//   "voltageC_v": 118.7,
+	avgAB := (res.VoltageAV + res.VoltageBV) / 2
+	if res.ContactorClosed && res.VoltageCV > avgAB / 2.1 && res.VoltageCV < avgAB / 1.9 {
```

Comment:
Simplify as

    math.Abs(res.VoltageCV - avgAB / 2) < 25 // V

Why do we need the `ContactorClosed` here- voltages should always apply?

Reply:
When contactor is open, api reports voltage on all three phases as ~2V. These are measuring the output. I used contactor state to avoid considering the workaround when it can't yet be determined. However, by using your simplified math, and comparing phase C to the supply voltage field, the logic still works.